src/alacris_lib/operator_contract.ml

```diff
@@ -23,13 +27,36 @@ let topic_of_amount (amnt : TokenAmount.t) : Bytes.t option =
 let topic_of_hash (hash : Digest.t) : Bytes.t option =
   Some (encode_function_parameters [abi_digest hash])
 
+(* Topics below correspond to events in the operator.sol code
+   Be careful of adjusting everything when you change the type like adding a balance.
+ *)
+
+let (topic_of_deposited: Bytes.t option) =
+  topic_of_hash (digest_of_string "Deposited(address,address,uint256,uint256)")
+
+let (topic_of_state_update: Bytes.t option) =
+  topic_of_hash (digest_of_string "StateUpdate(address,bytes32,uint64)")
+
+let (topic_of_claim_withdrawal: Bytes.t option) =
+  topic_of_hash (digest_of_string "ClaimWithdrawal(address,address,uint64,uint256,uint256,bytes32,uint64)")
+
+let (topic_of_withdraw: Bytes.t option) =
+  topic_of_hash (digest_of_string "Withdrawal(address,uint64,uint256,uint256,bytes32)")
+
+let (topic_of_rejected_claim_status: Bytes.t option) =
+  topic_of_hash (digest_of_string "RejectedClaimStatus(uint64)")
+
+let (topic_of_null_event: Bytes.t option) =
+  topic_of_hash (digest_of_string "NullEvent(bytes32)")
+
```

Comment:
I believe the latter two topics are not needed.

Reply:
I removed one single one, the one querying for the rejected status.